<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShakerGraph</class>
 <widget class="QWidget" name="ShakerGraph">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="Graph_Shaker" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>811</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="Graph_Sensor" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>811</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QCustomPlot" name="Graph_Doc" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>811</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>600</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>界面缩放</string>
   </property>
   <widget class="QRadioButton" name="zoom_h">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>横向伸缩</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="zoom_v">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>纵向伸缩</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="zoom_g">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>全局伸缩</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>600</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>X轴范围</string>
   </property>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* 滑动槽设置 */
QSlider::groove:horizontal {
	border:2px solid #ffffff;
	height:17px;
	background:qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #009688, stop:1 #cddc39);
	margin:2px 0;
}

/* 滑块设置 */
QSlider::handle:horizontal {
	background:qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #8bc34a, stop:1 #cddc39);
	border:2px solid #ffffff;
	border-radius:5px;
	width:18px;
	
	/* 滑块上下边与滑道重合 */
	margin:-2px 0;
}</string>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="L_hr">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 3px;
    padding: 0 8px;
    background: rgb(222, 222, 222);
    selection-background-color: darkgray;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>秒</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>600</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>信号选择</string>
   </property>
   <widget class="QCheckBox" name="en_H">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    image: url(:/icon/res/check_unchecked.png);
}

QCheckBox::indicator:checked {
    image: url(:/icon/res/check_checked.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>心跳通道</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>呼吸通道</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="en_B">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox {
    spacing: 5px;
}

QCheckBox::indicator {
    width: 20px;
    height: 20px;
}

QCheckBox::indicator:unchecked {
    image: url(:/icon/res/check_unchecked.png);
}

QCheckBox::indicator:checked {
    image: url(:/icon/res/check_checked.png);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">QCustomPlot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
